modules/payment/src/Event/PaymentEvents.php

```diff
@@ -10,7 +10,74 @@
    * @Event
    *
    * @see \Drupal\commerce_payment\Event\FilterPaymentGatewaysEvent
+   *
+   * @var string
    */
   const FILTER_PAYMENT_GATEWAYS = 'commerce_payment.filter_payment_gateways';
 
+
+  /**
+   * Name of the event fired after loading a payment.
+   *
+   * @Event
+   *
+   * @see \Drupal\commerce_payment\Event\PaymentEvent
+   *
+   * @var string
+   */
+  const PAYMENT_LOAD = 'commerce_payment.commerce_payment.load';
+
+  /**
+   * Name of the event fired before saving a payment.
+   *
+   * @Event
+   *
+   * @see \Drupal\commerce_payment\Event\PaymentEvent
+   *
+   * @var string
+   */
+  const PAYMENT_PRESAVE = 'commerce_payment.commerce_payment.presave';
+
+  /**
+   * Name of the event fired after saving a new payment.
+   *
+   * @Event
+   *
+   * @see \Drupal\commerce_payment\Event\PaymentEvent
+   */
+  const PAYMENT_INSERT = 'commerce_payment.commerce_payment.insert';
```

Comment:
I see an INSERT and a CREATE in product variation events. Do we need to duplicate the same here? The docs between the two aren't very clear about what the difference is.